token/services/identity/storage/kvs/storage.go

```diff
@@ -288,6 +290,10 @@ func NewInMemoryKVS() (KVS, error) {
 	return kvs.NewWithConfig(&mem.Driver{}, "", configService)
 }
 
+func NewHashicorpVaultKVS(client *vault.Client, path string) (KVS, error) {
```

Comment:
Please, remove this. It is not used anywhere and it introduces a dependency to the vault that is not needed.